src/analyses/ai.h

```diff
@@ -158,6 +162,17 @@ class ai_baset
     fixedpoint(goto_function.body, goto_functions, ns);
   }
 
+  /// Returns the abstract state before the given instruction
+  virtual const ai_domain_baset & abstract_state_before(
+    goto_programt::const_targett l) const = 0;
```

Comment:
`l` and `1` are almost indistinguishable. Maybe use `t`

Reply:
Fixed.

Reply:
Fixed.